Tasks/InstallAppleProvisioningProfile/preinstallprovprofile.ts

```diff
@@ -19,15 +19,16 @@ async function run() {
             let provProfilePath: string = await secureFileHelpers.downloadSecureFile(secureFileId);
 
             if (tl.exist(provProfilePath)) {
-                let UUID: string = await sign.getProvisioningProfileUUID(provProfilePath);
-                tl.setTaskVariable('APPLE_PROV_PROFILE_UUID', UUID);
+                const { provProfileUUID, provProfileName } = await sign.installProvisioningProfile(provProfilePath);
```

Comment:
You can't use this syntax in Node 5, that's why your tests are failing.

Reply:
Bumping the minAgentVersion in hopes that the node 5 tests stop running.

Reply:
Ehh, regardless of the minAgentVersion the test still runs against node5. Hopefully we retire node5 agents soon. For now I kept the response an object, but no longer destructure the response.

Reply:
Yeah, we might need to push a new base floor soon (for other reasons).